python/lsst/ctrl/orca/PegasusJobs.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+
+from __future__ import print_function
+from builtins import str
+from builtins import object
+import os
+import subprocess
+import re
+import time
+import lsst.log as log
+from lsst.ctrl.orca.CondorJobs import CondorJobs
+
+class PegasusJobs(CondorJobs):
+    """Handles interaction with Pegasus
+    This class is highly dependent on the output of the pegasus commands
+    """
+
+    def __init__(self):
+        log.debug("PegasusJobs:__init__")
+        return
+
+    def waitForJobToRun(self, num, extramsg=None):
```

Comment:
Those were meant to be deleted;  I was in the process of sub-classing CondorJobs.py, got distracted, and didn't end up deleting the rest of the methods that should have been deleted.  Thanks for catching that.